hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -44,44 +46,69 @@ function ActionLabelSettings() {
   return <>Action Label Settings coming here</>;
 }
 
-function WebhookActionerRender() {
-  return <Card>This is a card displaying a WebhookActionerRender</Card>;
+function WebhookActionerRender({url, headers, webhookType}) {
```

Comment:
This naming is unconventional to me. We see this in the "framework" that was used for SignalOsmosis configs... How about something like `WebhookActioner` or `WebhookActionerDetails`? It's the `Renderer` bit that doesn't feel right.

Reply:
Good with me